src/organisms/cards/PlaybackCard/index.js

```diff
@@ -14,10 +15,11 @@ function PlaybackCard(props) {
     onClick,
     icon,
     editLinkText,
+    marginBottom,
   } = props;
 
   return (
-    <div className={styles['playback-card']}>
+    <div className={classnames(styles['playback-card'], marginBottom && styles['add-margin'])}>
```

Comment:
Hmm -- how about a component that handles hiding for us? Either a wrapper or a HOC that can take a `hideOn` string prop: `hideOn='small large'`